playground/pages/[...slug].vue

```diff
@@ -0,0 +1,20 @@
+<script setup>
+const route = useRoute();
+
+const story = ref();
+try {
+  story.value = await useAsyncStoryblok(route.path, {
+    version: "draft",
+    language: "en",
+    resolve_relations: ["popular-articles.articles"]
+  });
+} catch (error) {
+  console.log(error);
```

Comment:
Shall we include some section in the documentation, maybe inside this one: https://github.com/storyblok/storyblok-nuxt#composition-api, about try catching the composable? What do you think @SebbeJohansson @alvarosabu ?

Reply:
I think that makes sense! Should probably mention something about why it differs from the non-async variant.

Reply:
For now, let's focus on the scope of this PR to just the error handling. Btw we already have what you mention in the API section: https://github.com/storyblok/storyblok-nuxt#api

Reply:
Hahaha fair enough 😁